src/awkward/operations/ak_from_rdataframe.py

```diff
@@ -1,18 +1,37 @@
 # BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
 
 import awkward as ak
+from awkward._nplikes.numpylike import NumpyMetadata
 
+np = NumpyMetadata.instance()
 
-def from_rdataframe(rdf, columns, offsets_type="int64_t", keep_order=False):
+
+def from_rdataframe(
+    rdf,
+    columns,
+    *,
+    keep_order=False,
+    offsets_type="int64",
+    highlevel=True,
+    behavior=None,
+    with_name=None,
+):
```

Comment:
```suggestion
def from_rdataframe(
    rdf,
    columns,
    *,
    keep_order=False,
    offsets_type="int64",
    with_name=None,
    highlevel=True,
    behavior=None,
):
```

because `highlevel` and `behavior` are generally last. Here's `ak.zip` as an example:

https://github.com/scikit-hep/awkward/blob/3535fbb8b4bc3443c09ba595620610fee8be0b74/src/awkward/operations/ak_zip.py#L9-L19